### ISSUE TITLE
FOV background method

### ISSUE THREAD

**Issue by cdeil** [gammapy/gammapy#5]
Implement a FOV background method (see [Berge 2007](http://adsabs.harvard.edu/abs/2007A%26A...466.1219B)) in [`gammapy/background/fov.py`](https://github.com/gammapy/gammapy/blob/master/gammapy/background/fov.py).

The main thing to implement is computing the 2D image from 1D radial acceptance curves, and then to scale to events outside exclusion regions and apply a FOV offset cut.

**Comment by cdeil** [comment]
@mapazarr – if you're looking for a challenge, you could start to implement this.

I haven't thought about the API much for `gammapy.background` yet, probably it's going to be a mix of classes (like [TablePSF](https://gammapy.readthedocs.org/en/latest/api/gammapy.irf.TablePSF.html) for example) and functions (like [ring_correlate](https://gammapy.readthedocs.org/en/latest/api/gammapy.image.ring_correlate.html), which by the way needs the Parameters and Returns section written in the docstring).

I think this utility function could be a good start:
```
def fill_acceptance_image(image, center, offset, acceptance)
```
where `image` is an [ImageHDU](http://astropy.readthedocs.org/en/latest/io/fits/api/images.html#astropy.io.fits.ImageHDU) object, center is a [SkyCoord](http://astropy.readthedocs.org/en/latest/api/astropy.coordinates.SkyCoord.html) and offset and acceptance and offset are 1-dimensional Numpy arrays and then the function returns the same `image` object, but with the `image.data` 2-dimensional numpy array filled.

For the unit test for that function you can get an `image` object by calling [make_empty_image](https://gammapy.readthedocs.org/en/latest/api/gammapy.image.make_empty_image.html) (which also has a mis-formatted docstring ... if you want to practice pull requests some more, small docstring fixes are a good way).

**Comment by cdeil** [comment]
This was implemented in #503 ... closing this (very old) issue now.
